src/rachanaGJun20/Array/PrimeNumberClass.java

```diff
@@ -0,0 +1,87 @@
+/*Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+
+package rachanaGJun20.Array;
+
+import java.util.Scanner;
+
+public class PrimeNumberClass {
+
+	void findIsNumberPrime(int number) {
+
+		boolean flag = true;
+		for (int i = 2; i <= number / 2; i++) {
+			if (number % i == 0) {
+				flag = false;
+				break;
+			}
+		}
+		if (flag == true)
+			System.out.println("Number is Prime\n");
+		else
+			System.out.println("Number is not Prime\n");
+	}
+
+	int findcountOfPrimeNumbers() {
+		boolean flag;
+		int primenumbercount = 0;
+		for (int num = 2; num <= 100; num++) {
+			flag = true;
+			for (int j = 2; j <= num / 2; j++) {
```

Comment:
Here you have to use findIsNumberPrime method to perform operation.

Reply:
yes got it will update....